lms/djangoapps/verify_student/views.py

```diff
@@ -1189,86 +1409,6 @@ def get(self, request, course_id, usage_id):
         }
         return render_to_response("verify_student/incourse_reverify.html", context)
 
-    @method_decorator(login_required)
-    def post(self, request, course_id, usage_id):
```

Comment:
Note to reviewers: this functionality has been folded into `SubmitPhotosView`, which now handles both the initial and re-verification cases.